### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by RashidMenhas**
### Use React components.
- This project will lay foundations for your Bookstore website. I created the React part of the app. - I also set up routing using React Router.
- Initialise React app.
- The building blocks of your app should be set as re-usable components. Create a directory for your components.
- Add [React Router](https://reactrouter.com/web/guides/quick-start) and set two <Route>and <Link> for the app's navigation:
- Books - the default view:
- Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no funcionality yet).
- Create a component called Book for displaying a single book (receive a title and an author as prop) and reuse it in a component that displays a list of books.
 - Should display a form for adding a book (no functionality yet).
- Create a separate component for this form (with inputs for a title and an author [IMPORTANT in the design you can see an input for a category - please replace it with an author.]).
Categories:
- Should display a button "Check status" only.

### CODE REVIEW

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @RashidMenhas, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._


- [ ] Kindly use the new template for the `Readme` file. Follow this [link](https://github.com/microverseinc/readme-template)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,247 @@
...
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```
Comment (zainsadaqat) [inline README.md:27]: ### [OPTIONAL]

- [ ] Please fix the Micorverse logo image in the README file 👍🏻 
Screenshot 2022-11-19 at 11 59 21 AM

**Review by zainsadaqat — APPROVED** [review]
# Pull Request Approved ✅ 🎊 🎉 
## Hello @RashidMenhas  👋 

### Your project is complete!  There is nothing else to say other than it's time to merge it.

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 ☑️ 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by RashidMenhas** [comment]
assalam o alikum sir thank you for approved my project. I have some Questions about microverse please help me Sent from my Verizon, Samsung Galaxy smartphone
-------- Original message --------From: Zain Sadaqat ***@***.***> Date: 11/19/22  12:00 PM  (GMT+05:00) To: RashidMenhas/books-store ***@***.***> Cc: Rashid Menhas ***@***.***>, Mention ***@***.***> Subject: Re: [RashidMenhas/books-store] Bookstore: initialize project with components (PR #1) 

@zainsadaqat approved this pull request.



Pull Request Approved ✅ 🎊 🎉

Hello @RashidMenhas  👋

Your project is complete!  There is nothing else to say other than it's time to merge it.

Congratulations!!! 🎉 🥳 👏 🙌 🎊 ☑️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.



As described in the Code reviews limits policy  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.



In README.md:

> +  - Built With

+  - Live Demo

+- Getting Started

+- Authors

+- Future Features

+- Contributing

+- Show your support

+- Acknowledgements

+- License

+

+After you're finished please remove all the comments and instructions!

+-->

+

+<div align="center">

+

+  logo



[OPTIONAL]



 Please fix the Micorverse logo image in the README file 👍🏻







—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

[

{

***@***.***": "http://schema.org",

***@***.***": "EmailMessage",

"potentialAction": {

***@***.***": "ViewAction",

"target": "https://github.com/RashidMenhas/books-store/pull/1#pullrequestreview-1187030836",

"url": "https://github.com/RashidMenhas/books-store/pull/1#pullrequestreview-1187030836",

"name": "View Pull Request"

},

"description": "View this Pull Request on GitHub",

"publisher": {

***@***.***": "Organization",

"name": "GitHub",

"url": "https://github.com"

}

}

]

**Pull request merged**